build.ps1

```diff
@@ -3,46 +3,29 @@
 [CmdletBinding(PositionalBinding = $false)]
 Param(
   [Parameter(ValueFromRemainingArguments = $true)][String[]]$verbs,
-  [Switch]$reset,
   [Switch]$prereqs
 )
 
+$upstream_repo_path = Join-Path -Path ".." -ChildPath "playwright"
+
 function Get-Help() {
-  Write-Host "Actions (defaults to help):"
+  Write-Host "Commands:"
   Write-Host "  init                      Performs the initalization of the repository, and installs/downloads"
-  Write-Host "                            all the required artifacts (i.e. driver), and initializes the submodule."
-  Write-Host "                            -reset     Forces the submodule to be updated."
+  Write-Host "                            all the required artifacts (i.e. driver)."
   Write-Host "                            -prereqs   Installs all the required prerequisites and dev certificates."
   Write-Host ""
-  Write-Host "  roll [commit sha]         Moves the submodule to the commit specified, generates the API, and downloads"
-  Write-Host "                            downloads the new driver."
+  Write-Host "  roll <driver-version>     Updates the API version, downloads driver, generates the API, and transport interfaces."
```

Comment:
I don't see where we call generate_dotnet_channels here.

Reply:
fixed